src/main/java/com/example/rentalSystem/domain/student/dto/request/StudentSignUpRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.example.rentalSystem.domain.student.dto.request;
+
+import com.example.rentalSystem.domain.student.entity.Student;
+import java.time.LocalDateTime;
+import lombok.Builder;
+
+public record StudentSignUpRequest(String name, String studentNumber,String password, String email, String affiliation, String loginId,String major, String phoneNumber) {
```

Comment:
가독성이 떨어져서, 변수 하나당 모두 줄바꿈 해주시는 게 좋을 것 같아요
```suggestion
public record StudentSignUpRequest(
    String name, 
    String studentNumber,
    String password, 
    String email, 
    String affiliation, 
    String loginId,
    String major, 
    String phoneNumber
) {
```